contracts/serverless-v2/Relay.sol

```diff
@@ -40,10 +40,11 @@ contract Relay is
      * @param maxAge The maximum age for attestations.
      * @param _token The ERC20 token used for payments and deposits.
      * @param _globalMinTimeout The minimum timeout value for jobs.
-     * @param _globalMaxTimeout The maximum timeout value for jobs.
-     * @param _overallTimeout The overall timeout value for job execution.
-     * @param _executionFeePerMs The fee per millisecond for job execution.
-     * @param _gatewayFeePerJob The fixed fee per job for the gateway.
+     * @param _globalMaxTimeout The maximum timeout value for jobs. This refers to the max time for the executor to 
```

Comment:
again, it's wrong, it's just time of execution on executor, doesn't include the response time on common chian.

Reply:
got it